registration/views.py

```diff
@@ -23,6 +29,31 @@ def get_serializer_class(self):
             return FamilyDetailSerializer
         return FamilySerializer
 
+    @action(
+        methods=["get"],
+        detail=False,
+        permission_classes=[permissions.IsAuthenticated],
+        url_path="search",
+        url_name="search",
+    )
+    def search(self, request):
+        first_name = self.request.query_params.get("first_name")
+        last_name = self.request.query_params.get("last_name")
+        if not first_name and not last_name:
+            return Response(
+                "No search parameters found.", status=status.HTTP_400_BAD_REQUEST
+            )
+        result = Family.objects
+        if first_name:
+            result = result.filter(parent__first_name__iexact=first_name)
+        if last_name:
+            result = result.filter(parent__last_name__iexact=last_name)
+
+        # how granular / accepting is our search, do we want to be able to search by parts of first names / last names ?
```

Comment:
I don't think we need to be able to do fuzzy search for the MVP. You could always ask design folks.